tests/vmi_multus_test.go

```diff
@@ -587,25 +609,9 @@ var _ = Describe("SRIOV", func() {
 			Skip("Sriov is not enabled in this environment. Skip these tests using - export FUNC_TEST_ARGS='--ginkgo.skip=SRIOV'")
 		}
 
-		// Create two sriov networks referring to the same resource name
-		result := virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov")).
-			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
-		result = virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov2")).
-			Body([]byte(fmt.Sprintf(sriovConfCRD, "sriov2", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
-		result = virtClient.RestClient().
-			Post().
-			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov-link-enabled")).
-			Body([]byte(fmt.Sprintf(sriovLinkEnableConfCRD, "sriov-link-enabled", tests.NamespaceTestDefault, sriovResourceName))).
-			Do()
-		Expect(result.Error()).NotTo(HaveOccurred())
+		Expect(createNetwork("sriov", tests.NamespaceTestDefault, sriovConfCRD)).To((Succeed()))
```

Comment:
Could you please add Expect error messages?

Reply:
can you elaborate?
if the Expect will fail it will include the error returning from `createNetwork(..)` which suppose to provide the needed info.
anything else you would like to add?

Reply:
By Expect error message I mean to a string that describe what should have happen but faild at the function we use on the `Expect` returned value
So it should be like:
```
Expect( foo()).To(BeTrue(), <what should have happen but failed>)
```

In this case:
```
Expect(createNetwork("sriov", tests.NamespaceTestDefault, sriovConfCRD)).To((Succeed()), fmt.Sprintf("should successfully delete NetworkAttachmentDefinition '%s'", "sriov"))
```

Reply:
The error needs to include all the needed info, if it does not it needs to be added there.
Adding text on the assert is bad and whenever possible I do not want to see it.

Reply:
Same as https://github.com/kubevirt/kubevirt/pull/3479/commits/fab84ff96ef1761f7f33844ee09965747f1a0d9c#r468469079
since the error will include the object we are trying to delete I don't Think it should be included in the formatting.
I addd the "should have.." part.  What do you think?